packages/code-studio/src/main/AppRouter.tsx

```diff
@@ -7,12 +7,16 @@ import {
 } from 'react-router-dom';
 import AppInit from './AppInit';
 
+const basename = new URL(document.baseURI).pathname;
 function AppRouter(): ReactElement {
   return (
-    <Router basename={import.meta.env.BASE_URL}>
+    <Router basename={basename}>
       <Switch>
         <Route exact path="/" component={AppInit} />
-        <Route path="/notebook/:notebookPath+" component={AppInit} />
+        <Route
+          path={`/${import.meta.env.VITE_ROUTE_NOTEBOOKS}:notebookPath+`}
+          component={AppInit}
+        />
```

Comment:
No hash router? I don't think this will work without jetty rewriting the base tag

It works for me in dev mode, but not production

Reply:
No hash router... I didn't like it always putting a hash in the URL. Also trickier to resolve paths...